script/1_0_2current_en_script/Ciel/Day 4/04_01_CIEL04_4_2.txt

```diff
@@ -99,4 +99,4 @@ Arcueid smiles and holds out her left hand.
 <Page29>
 "...Yeah, this really is insane."
 But there's nothing I can do about it.
-I reluctantly hold out my left hand and shake hands with the girl in white who calls herself a vampire.
+I reluctantly hold out my left hand and shake hands with the woman in white who calls herself a vampire.
```

Comment:
this is a repeated line too

Reply:
Seems to match the line it's a copy of though?
```
ross@mjolnir:/h/r/g/Tsukihime-Translation$ gg 'shake hands'
script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_4_1.txt:asks me to shake hands with her.
script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_4_1.txt:I reluctantly hold out my left hand and shake hands with the woman in white who calls herself a vampire.
script/1_0_2current_en_script/Ciel/Day 4/04_01_CIEL04_4_2.txt:asks me to shake hands with her.
script/1_0_2current_en_script/Ciel/Day 4/04_01_CIEL04_4_2.txt:I reluctantly hold out my left hand and shake hands with the woman in white who calls herself a vampire.
```